### ISSUE TITLE
Specify custom function for onClose

### ISSUE THREAD

**Issue by GCDeveloper** [GA-MO/react-confirm-alert#21]
There currently seems to be no way to specify a custom onClose function, i.e. when click outside the modal, and you want to do some custom logic when it closes.

**Comment by GA-MO** [comment]
@GCDeveloper 
You can use `willUnmount` for handle some logic when it closes.
```js
   confirmAlert({
      willUnmount: () => // Some logic,
      customUI: ({ onClose }) => {
        return (
          <div className='custom-ui'>
            <h1>Are you sure?</h1>
            <p>You want to delete this file?</p>
            <button>No</button>
            <button>Yes, Delete it!</button>
          </div>
        )
      }
    })
```

**Comment by GA-MO** [comment]
This issue was solved in version 2.3.0
